game.js

```diff
@@ -927,6 +927,7 @@ const _gameUpdate = (timestamp, timeDiff) => {
           const result = metaversefileApi.getPairByPhysicsId(collisionId);
           if (result) {
             const [app, physicsObject] = result;
+            if (app.npcPlayer?.avatar?.ragdoll) return; // npcPlayer already die and in ragdoll animation.
```

Comment:
Early return in a large function is bad. This logic should be refactored to not need it.